src/models/EnrollUser.js

```diff
@@ -0,0 +1,75 @@
+/*!
+ * Copyright (c) 2019, Okta, Inc. and/or its affiliates. All rights reserved.
+ * The Okta software accompanied by this notice is provided pursuant to the Apache License, Version 2.0 (the 'License.")
+ *
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0.
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *
+ * See the License for the specific language governing permissions and limitations under the License.
+ */
+
+define([
+  'okta',
+  'models/ProfileSchema',
+  './BaseLoginModel'
+],
+function (Okta, ProfileSchema, BaseLoginModel) {
+  var {_} = Okta;
+
+  return BaseLoginModel.extend({
+    props: this.properties,
```

Comment:
I doubt this is necessary (or work properly)?
`this` refers to the context of callback function of `define` hence `this.properties` is undefined.

Reply:
not necessary